### PR TITLE
Add keyboards/whitefox/keymaps/constexpr

### PR DESCRIPTION
**Pull request by vomindoraan**
My WhiteFox/NightFox keymap.

I used my GeekHack username as I don't really go by vomindoraan in the keyboard community. If it's customary to use GitHub usernames, let me know and I'll rename the folder.

### CODE REVIEW

File: keyboards/whitefox/keymaps/constexpr/rules.mk [hunk 1]
```diff
@@ -0,0 +1,8 @@
+BOOTMAGIC_ENABLE = no
+CONSOLE_ENABLE = yes
+TAP_DANCE_ENABLE = yes
+UNICODE_ENABLE = yes
+
+ifndef QUANTUM_DIR
```
Comment (drashna) [inline keyboards/whitefox/keymaps/constexpr/rules.mk]: This block is a hold over from an older makefile configuration.  If you could remove it?

File: keyboards/whitefox/keymaps/constexpr/config.h [hunk 2]
```diff
@@ -0,0 +1,19 @@
+#ifndef CONFIG_USER_H
```
Comment (drashna) [inline keyboards/whitefox/keymaps/constexpr/config.h]: Could you remove the first three lines and the `#endif` at the end and use just `#pragma once`

Eg, remove this:
```c
#ifndef CONFIG_USER_H
#define CONFIG_USER_H

#include "../../config.h"
```
And the `#endif` at the end.

**Comment by drashna** [comment]
And the username is fine.  Using the github account name is just a recommendation, not a requirement.

File: keyboards/whitefox/keymaps/constexpr/rules.mk (hunk 1 above)
Comment (vomindoraan) [reply keyboards/whitefox/keymaps/constexpr/rules.mk]: I was wondering about this. Sure thing, I'll do that right away.

**Comment by vomindoraan** [comment]
@drashna I've changed the files as requested.

**Comment by drashna** [comment]
Awesome, thanks!

**Pull request merged**